spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamDefinition.java

```diff
@@ -127,25 +127,21 @@ public String toString() {
 
 
 	/**
-	 * Iterator that prevents mutation of its backing data structure.
-	 *
-	 * @param <T> the type of elements returned by this iterator
+	 * Iterator that walks through ModuleDefinitions in deployment order, that is, from downstream (sink, if present)
+	 * to upstream (source) module.
```

Comment:
This javadoc would not be accurate in the case of named channels (at least not on the "source" end). For sink it does say "if present", but perhaps we should clarify the case where it would not be?